docs/Generic_easyblocks.rst

```diff
@@ -0,0 +1,781 @@
+==============================
```

Comment:
include a clear comment on top that this is auto-generated (let the script do that)